evap/results/exporters.py

```diff
@@ -284,3 +284,45 @@ def _write(exporter, label, style, rows, cols):
         exporter.col += cols - 1
     else:
         exporter.sheet.write(exporter.row, exporter.col, label, style)
+
+
+class TextAnswerExcelExporter:
+    # pylint: disable=too-many-instance-attributes
+
+    def __init__(self, evaluation_name, semester_name, responsibles, results):
+        self.evaluation_name = evaluation_name
+        self.semester_name = semester_name
+        self.responsibles = responsibles
+        self.results = results
+
+        self.styles = {
+            "default": xlwt.Style.default_style,
+            "border_top": xlwt.easyxf("borders: top medium"),
+        }
+
+        self.workbook = xlwt.Workbook()
+        self.sheet = self.workbook.add_sheet(_("Text Answers"))
+        self.row = 0
+        self.col = 0
+        self.sheet.col(0).width = 10000
+        self.sheet.col(1).width = 40000
+
+    def export(self, response):
+        writec(self, self.evaluation_name, "default")
```

Comment:
this should be the `full_name` of the evaluation

Reply:
The view calls this with
```python
TextAnswerExcelExporter(evaluation.name, evaluation.course.semester.name,
                            evaluation.course.responsibles_names, results).export(response)
```
and Semester only has `name` and `short_name` , right?

Reply:
Nevermind, caller provides the `full_name` now